home/templates/accounts/sign-in.html

```diff
@@ -9,71 +9,86 @@
-                                    </div>  
-                                </div>
-                                <span class="text-danger"> {{ field.errors }} </span>
+                        <form method="post" class="mt-4">
+                            {% csrf_token %}
+                            <!-- Form -->
+                            {% if form.non_field_errors %}
+                                {% for error in form.non_field_errors %}
+                                    <span class="text-danger"> {{ error }} </span>
                                 {% endfor %}
-                                <!-- End of Form -->
-                                <div class="d-flex justify-content-between align-items-center mb-4">
-                                    <div class="form-check mb-0">
-                                        <input class="form-check-input" type="checkbox" value="" id="remember">
-                                        <label class="form-check-label mb-0" for="remember">
-                                            Remember me
-                                        </label>
-                                    </div>
-                                    <div><a href="{% url 'password_reset' %}" class="small text-right lost-password">Lost password?</a></div>
-                                </div>
-                                <div class="d-grid">
-                                    <button type="submit" class="btn btn-warning btn-primary">Sign in</button>
+                            {% endif %}
+                            {% for field in form %}
+                            <div class="form-group mb-4">
+                                <label for="{{ field.id_for_label }}">{{ field.label }}</label>
+                                <div class="input-group">
+                                    <span class="input-group-text" id="basic-addon1">
+                                        <span class="fas {% if field.name == 'username' %} fa-user {% else %} fa-unlock-alt {% endif %}"></span>
+                                    </span>
+                                    {{ field }}
+                                </div>  
+                                <span class="text-danger"> {{ field.errors }} </span>
+                            </div>
+                            {% endfor %}
+                            <!-- End of Form -->
+
+                            <!-- 2FA Input -->
+                            <div class="form-group mb-4">
+                                <label for="2fa-code">2FA Code</label>
+                                <div class="input-group">
+                                    <span class="input-group-text" id="basic-addon2">
+                                        <span class="fas fa-key"></span>
+                                    </span>
+                                    <input type="text" id="2fa-code" name="2fa_code" class="form-control" placeholder="Enter your 2FA code">

```

Comment:
the site is unable to actually send 2FA codes in a development environment so this prevents logging in
could you fix this, perhaps print the 2FA code to the terminal when DEBUG=true